game.js

```diff
@@ -42,22 +56,36 @@ function loop() {
         shipSpeed = 0;
         shipPosition = 0;
     }
-    layout();
+    if(shipSpeed >0 &&
+        shipPosition > 0)
+        {
+            layout();
+        }
+        else{
+            document.getElementById('fuel').innerHTML = 'UFO Crashed';
+
+        }
+    
 }
 
 function reset() {
     shipPosition = 1;
     shipSpeed = 0;
     fuel=maxFuel;    
+    
     layout();
+
 }
 
 function startThrust() {
     thrust = true;  
+    initFlame();
```

Comment:
It feels like initFlame() should be in the init() method and then have another showflame() and hideFlame() method to use here